spec.html

```diff
@@ -1554,7 +1554,7 @@ <h2>[[GetOwnProperty]] (_P_)</h2>
             The Type of the return value must be either Property Descriptor or Undefined.
           </li>
           <li>
-            If the Type of the return value is Property Descriptor, the return value must be a complete property descriptor (see <emu-xref href="#sec-completepropertydescriptor"></emu-xref>).
+            If the Type of the return value is Property Descriptor, the return value must be a fully populated Property Descriptor (see <emu-xref href="#sec-completepropertydescriptor"></emu-xref>).
```

Comment:
Since the section is named “complete”, should we not go with that for the wording?

Reply:
I opted to use the term as already defined in the section defining Property Descriptors ([`sec-property-descriptor-specification-type`](https://tc39.github.io/ecma262/#sec-property-descriptor-specification-type)). Swapping all to be "complete Property Descriptors" is also ok with me, a similar number of changed lines are required.

Reply:
Fair - https://tc39.github.io/ecma262/#sec-completepropertydescriptor seems to me the stronger precedent. Let’s see what others think.

Reply:
On the other hand, we may wish to avoid possible confusion between complete (verb) and complete (adjective).

Reply:
“completed” for the adjective case would work fine.

Reply:
I think the link to "CompletePropertyDescriptor" in this context is a bit odd; I would expect a definition rather than an algorithm.

I like the change to "fully populated...". Can we just remove the link to the algorithm?